src/Microsoft.DotNet.Interactive.CSharpProject/RoslynWorkspaceUtilities/BuildCacheFileUtilities.cs

```diff
@@ -78,30 +78,40 @@ internal static class BuildCacheFileUtilities
 </Project>
 """;
 
-    internal static async Task<string> BuildAndCreateCacheFileAsync(string csprojFilePath)
+    internal static async Task BuildAndCreateCacheFileAsync(string csprojFilePath)
     {
+        if (string.IsNullOrEmpty(csprojFilePath) || !File.Exists(csprojFilePath))
```

Comment:
I would make these separate conditions. If null is passed, then that's a different category of error, indicating a bug in this codebase, whereas the path not existing is likely a deployment or environment problem.